RazorGenerator.Testing/WebViewPageExtensions.cs

```diff
@@ -24,28 +24,35 @@ public static class WebViewPageExtensions
 
         private static readonly Dictionary<string, ViewPlaceholder> _views = new Dictionary<string, ViewPlaceholder>();
 
-        public static IViewEngine ViewEngine
+        private static StringWriter _writer;
+
+        public static DummyViewEngine ViewEngine
         {
             get { return _viewEngine; }
         }
 
+        public static StringWriter Writer
```

Comment:
Might do. I'll have to run some more tests to see if I really need it. Exposed in order to write partials. Can it be declared unsafe for parallel execution?